docs/maintainer-guide/governance.md

```diff
@@ -37,49 +37,94 @@ To become a Committer:
 
-Reviewers have additional responsibilities over and above those of a Committer. These responsibilities ensure the smooth running of the project. Reviewers are expected to review code contributions, approve changes to this document, and manage the copyrights within the project outputs.
+The ESLint project is jointly governed by a Technical Steering Committee (TSC) which is responsible for high-level guidance of the project.
 
-Reviewers fulfill all requirements of Committers, and also:
+The TSC has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+
+TSC seats are not time-limited. There is no fixed size of the TSC. The TSC should be of such a size as to ensure adequate coverage of important areas of expertise balanced with the ability to make decisions efficiently.
+
+The TSC may add additional members to the TSC by a standard TSC motion.
+
+A TSC member may be removed from the TSC by voluntary resignation, or by a standard TSC motion.
+
+Changes to TSC membership should be posted in the agenda, and may be suggested as any other agenda item (see "TSC Meetings" below).
+
+No more than 1/3 of the TSC members may be affiliated with the same employer. If removal or resignation of a TSC member, or a change of employment by a TSC member, creates a situation where more than 1/3 of the TSC membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more TSC members affiliated with the over-represented employer(s).
 
-* Contributions can be reviewed by other Reviewers, but this is not explicitly required.
-* May merge external pull requests for accepted issues upon reviewing and approving the changes.
-* May merge their own pull requests once they have collected the feedback they deem necessary. (No pull request should be merged without at least one Committer/Reviewer comment stating they've looked at the code.)
 
+TSC members have additional responsibilities over and above those of a Committer. These responsibilities ensure the smooth running of the project. TSC members are expected to review code contributions, approve changes to this document, manage the copyrights within the project outputs, and attend regular TSC meetings.
 
-To become a Reviewer:
+TSC members fulfill all requirements of Committers, and also:
 
-* Reviewers are expected to work in a helpful and collaborative way with the community.
+* May merge external pull requests for accepted issues upon reviewing and approving the changes.
+* May merge their own pull requests once they have collected the feedback they deem necessary. (No pull request should be merged without at least one Committer/TSC member comment stating they've looked at the code.)
+
+To become a TSC member:
+
+* TSC members are expected to work in a helpful and collaborative way with the community.
```

Comment:
Small parallelism issue here: This new point says "TSC members are" (include subject), but subsequent points start with the verb and have implicit subject "TSC members".

My suggestion is to rework line 75 to say "In order to be considered for TSC membership, contributors:" and then remove "TSC members" from this line. But there are a number of ways to fix this.